source/locales/translation.ts

```diff
@@ -0,0 +1,121 @@
+/*
+	This module contains all types and functions related to the translation.
+*/
+
+import { i18n } from 'i18next'
+
+import uiEn from './ui/ui-en.yaml'
+import uiEs from './ui/ui-es.yaml'
+import uiFr from './ui/ui-fr.yaml'
+import uiIt from './ui/ui-it.yaml'
+
+import faqEn from './faq/FAQ-en-us.yaml'
+import faqEs from './faq/FAQ-es.yaml'
+import faqFr from './faq/FAQ-fr.yaml'
```

Comment:
General question : @EmileRolley have you thought about placing a link on every page of NGC, or at least as a start on pages such as the FAQ or releases that directs the user to the relevant markdown github page for him to edit a typo ? 

Not necessary for this release though

Reply:
But I'm thinking all these automatic translations will lead to lots of needed improvements, and we'll need to catch them in constructive edits rather than build frustration :)

Reply:
Yes I think it will be needed when an other language than Fr is selected :+1: